app/tests/e2e/issuer/test_issuer_ld_bbs.py

```diff
@@ -0,0 +1,382 @@
+import pytest
+from aries_cloudcontroller import (
+    AcaPyClient,
+    Credential,
+    LDProofVCDetail,
+    LDProofVCDetailOptions,
+)
+from assertpy import assert_that
+from fastapi import HTTPException
+
+from app.models.issuer import SendCredential
+from app.routes.issuer import router as issuer_router
+from app.routes.oob import router as oob_router
+from app.routes.wallet import router as wallet_router
+from app.tests.util.ecosystem_connections import FaberAliceConnect
+from app.tests.util.trust_registry import DidKey
+from app.tests.util.webhooks import check_webhook_state
+from shared import RichAsyncClient
+
+CREDENTIALS_BASE_PATH = issuer_router.prefix
+OOB_BASE_PATH = oob_router.prefix
+WALLET = wallet_router.prefix
+
+# This is the json of the below credential
+# {
+#     "type": "ld_proof",
+#     "connection_id": "",
+#     "protocol_version": "v2",
+#     "ld_credential_detail": {
+#         "credential": {
+#             "@context": [
+#                 "https://www.w3.org/2018/credentials/v1",
+#                 "https://www.w3.org/2018/credentials/examples/v1",
+#             ],
+#             "type": ["VerifiableCredential", "UniversityDegreeCredential"],
+#             "credentialSubject": {
+#                 "degree": {
+#                     "type": "BachelorDegree",
+#                     "name": "Bachelor of Science and Arts",
+#                 },
+#                 "college": "Faber College",
+#             },
+#             "issuanceDate": "2021-04-12",
+#             "issuer": "",
+#         },
+#         "options": "",
+#     },
+# }
+
+credential = SendCredential(
+    type="ld_proof",
+    connection_id="",
+    protocol_version="v2",
+    ld_credential_detail=LDProofVCDetail(
+        credential=Credential(
+            context=[
+                "https://www.w3.org/2018/credentials/v1",
+                "https://www.w3.org/2018/credentials/examples/v1",
+            ],
+            type=["VerifiableCredential", "UniversityDegreeCredential"],
+            credentialSubject={
+                "degree": {
+                    "type": "BachelorDegree",
+                    "name": "Bachelor of Science and Arts",
+                },
+                "college": "Faber College",
+            },
+            issuanceDate="2021-04-12",
+            issuer="",
+        ),
+        options=LDProofVCDetailOptions(proofType=""),
+    ),
+).dict(by_alias=True, exclude_unset=True)
+
+
+@pytest.mark.anyio
+@pytest.mark.parametrize("register_key_issuer", ["bls12381g2"], indirect=True)
```

Comment:
I think for readability it'll be better to have 2 separate fixtures: `register_issuer_did_key_bbs` and `register_issuer_did_key_ed25519`. Then we can have two dataclasses to represent the fixture types: DidKeyBbs and DidKeyEd25519.

At the moment, we have to scroll up to this parametrize method to see which signature type is being used. And it'll be clearer in the code to see `did = register_issuer_did_key_bbs`